### DISCUSSION TITLE
Carousel of models not working. Problem with switcSrc

### DISCUSSION THREAD

**Discussion by Hexa93** [google/model-viewer#4092] [Q&A]
Hello everyone. I'm having a hard time trying to find out why the file directory won't work. I've followed [this example](https://modelviewer.dev/examples/augmentedreality/index.html) with a few tweaks. My problem is with the slider buttons at the bottom, when I try to change the file, the site freeze (i.e. switchSrc doesn't switch the src).
I've used this function:

 window.switchSrc = (element, name) => {
    const base = "C:/Users/User/Desktop/products" + name;
    modelViewer.src = base + '.glb';
    modelViewer.poster = base + '.webp';
    const slides = document.querySelectorAll(".slide");
    slides.forEach((element) => {element.classList.remove("selected");});
    element.classList.add("selected");
  };

As well as this one:

const baseURL = "C:/Users/User/Desktop/products/";

window.switchSrc = (element, name) => {
  const src = baseURL + name + '.glb';
  const poster = baseURL + name + '.webp';
  modelViewer.src = src;
  modelViewer.poster = poster;
  const slides = document.querySelectorAll(".slide");
  slides.forEach((element) => {element.classList.remove("selected");});
  element.classList.add("selected");
};

Neither seems to work. It is not reading the directory correctly.

Thank you in advance for any further help.

**Comment by WadeMorrison** [comment]
Hey there, I'm not sure if this will work or not, but I did notice one change from yours and the example: 

In your top example I see the path doesn't end with a '/', where as the example it does. Meaning it's adding the 'name' to the very end of that directory which most likely doesn't exist?
So it should be:

const base = "C:/Users/User/Desktop/products/" + name;

Hope that helps!

**Comment by elalish** [comment]
@Hexa93 Please don't cross-post - it makes it harder to respond to you and splits the conversation.
